src/ocm/components/clusterDetail/CancelInstallationModal.tsx

```diff
@@ -66,7 +66,9 @@ const CancelInstallationModal: React.FC = () => {
         <Button
           key="submit"
           variant={ButtonVariant.danger}
-          onClick={handleSubmit}
+          onClick={() => {
+            void handleSubmit();
```

Comment:
I'd revert this unless we test this case specifically

Reply:
This change is safe, imo